<def f='boost-1.85.0/libs/throw_exception/include/boost/throw_exception.hpp' l='86' ll='88' type='void boost::wrapexcept::copy_from(const void * )'/>
